MAUI/ListView/swiping.md

```diff
@@ -0,0 +1,563 @@
+        <Grid VerticalOptions="Center" HorizontalOptions="Center">
+          <Image Grid.Column="0"
+                 Grid.Row="0"
+                 HeightRequest="35"
+                 WidthRequest="35"
+                 BackgroundColor="Transparent"
+                 Source="Delete.png" />
+        </Grid>
+      </Grid>
+    </DataTemplate>
+  </syncfusion:SfListView.EndSwipeTemplate>
+ </syncfusion:SfListView>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+listView.AllowSwiping = true;
+listView.SelectionMode = SelectionMode.None;
+listView.SwipeOffset = 360;
+listView.SwipeThreshold = 30;
+listView.SwipeStarting += ListView_SwipeStarting;
+listView.SwipeEnded += ListView_SwipeEnded;
+listView.Swiping += ListView_Swiping;
+listView.EndSwipeTemplate = new DataTemplate(() =>
+{
+   var grid = new Grid();
+
+   var grid1 = new Grid()
+   {
+    BackgroundColor = Colors.FromHex("#DC595F"),
+    HorizontalOptions = LayoutOptions.Fill,
+    VerticalOptions = LayoutOptions.Fill
+   };
+   var deleteGrid = new Grid() { HorizontalOptions = LayoutOptions.Center, VerticalOptions = LayoutOptions.Center };
+   var deleteImage = new Image() { BackgroundColor = Colors.Transparent, HeightRequest = 35, WidthRequest = 35 };
+   deleteImage.Source = ImageSource.FromResource("Swiping.Images.Delete.png");
+   deleteGrid.Children.Add(deleteImage);
+   grid1.Children.Add(deleteGrid);
+
+   grid.Children.Add(grid1);
+
+   return grid;
+});
+
+private void ListView_SwipeEnded(object sender, SwipeEndedEventArgs e)
+{
+  if (e.Offset >= 360)
+  {
+     viewModel.InboxInfo.RemoveAt(e.Index);
+     listView.ResetSwipeItem();
+  }
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Programmatic swiping
+
+Using the `SwipeItem` method, you can swipe an item programmatically based on the given offset value. You need to pass the item to be swiped and `SwipeOffset` as a parameter in the `SwipeItem` method.
+
+<table>
+<tr>
+<th>Parameters</th>
+<th>Description</th>
+</tr>
+<tr>
+<td>item data</td>
+<td>Represents the item data of list view item.</td>
+</tr>
+<tr>
+<td>offset</td>
+<td>Represents the value that how much the item need to swipe.</td>
+</tr>
+</table>
+
+The `SwipeOffset` value should be positive for the left swiping of the listview item.
+ 
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void LeftSwipeButton_Clicked(object sender, EventArgs e)
+{
+    ListView.SwipeItem(viewModel.contactsinfo[1], 200);
+}
+{% endhighlight%}
+{% endtabs %}
+
+The `SwipeOffset` value should be negative for the right-swiping of the listview item.
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void RightSwipeButton_Clicked(object sender, EventArgs e)
+{
+    ListView.SwipeItem(viewModel.contactsinfo[1], -200);
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Programmatic Start and End swiping in listview](Images/swiping/maui-listview-programmatic-swiping.gif)
```

Comment:
add .NET MAUI ListView wherever ListView used in alt text

Reply:
Modified